SeleniumTrainingCenter/Pages/GmailPage.cs

```diff
@@ -0,0 +1,44 @@
+﻿using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+
+namespace SeleniumTrainingCenter.Pages
+{
+    public class GmailPage : MailPage
+    {
+        public GmailPage(IWebDriver driver, string url) : base(driver, url)
+        {
+        }
+
+        public override GmailPage Login(string email, string password)
+        {
+            string EMAIL_INPUT = "//input[@class='whsOnd zHQkBf' and @type='email']";
```

Comment:
Move your locators at the class level to extend abilities to use them within different methods just in case.